CHANGELOG.md

```diff
@@ -7,9 +7,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Empty test cases
 - Added a bootstrap file for the tests
 - Setup for code coverage
-
+- added created example pdf files to .gitignore
```

Comment:
Also please move this to changed, as it is not an "added" feature but a "changed" behavior